### DISCUSSION TITLE
PTZ Cam ‘Off’

### DISCUSSION THREAD

**Discussion by cmb991** [dgreif/ring#1545] [Q&A]
I have a Ring Pan-Tilt indoor camera and it just says ‘This camera is in “Off” mode and cannot stream or record.’
In HomeKit, it says 
Unknown Model (stickup_cam_mini_ptz_v1)

Is there any fix for this?

**Comment by tsightler** [comment]
If you are seeing this message in Homekit I think you are seeing a long known Homekit bug, which, if you Google, you will find dozens of similar reports with pretty much all kinds of cameras and various workarounds to get it going again.  Sometimes restarting the Homekit hub can help, but usually I've found that you must remove and re-add the camera.  There's nothing in the plugin that can even tell Homekit that a device is "on" or "off" so I'm pretty sure it is not coming from the plugin.

Here's a similar report from Startling hub:

https://support.starlinghome.io/hc/en-us/articles/18305397169175-Cameras-stuck-in-off-mode-in-Apple-Home#:~:text=Go%20to%20the%20Apple%20Home,then%20plug%20power%20back%20in

Also, you may need to make sure you don't have any Homekit Secure Video settings configured, as this is not supported by this plugin.

**Comment by cmb991** [comment]
That worked, thank you!
